app/src/behaviors/behavior_sticky_key.c

```diff
@@ -164,12 +166,12 @@ static int on_sticky_key_binding_released(struct zmk_behavior_binding *binding,
 
     // No other key was pressed. Start the timer.
     sticky_key->timer_started = true;
-    sticky_key->release_at = event.timestamp + sticky_key->config->release_after_ms;
     // adjust timer in case this behavior was queued by a hold-tap
-    int32_t ms_left = sticky_key->release_at - k_uptime_get();
```

Comment:
I don't see how this is accounting for the comment above about adjusting for being held.. The timestamp in the event might be "older" than the actual time if the release was captured then re-released.

Did you test this in combination of hold taps?

Reply:
I included this change by mistake. I wanted to test if it would break.

> Did you test this in combination of hold taps?

I planned to, but I haven't had the chance to test it yet.